src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1013,7 +1034,31 @@ private CorInfoType asCorInfoType(CORINFO_CLASS_STRUCT_* cls)
         }
 
         private int appendClassName(short** ppBuf, ref int pnBufLen, CORINFO_CLASS_STRUCT_* cls, bool fNamespace, bool fFullInst, bool fAssembly)
-        { throw new NotImplementedException("appendClassName"); }
+        {
+            // We support enough of this to make SIMD work, but not much else.
+
+            // TODO: figure out why the marshalling for fFullInst and fAssembly is wrong.
```

Comment:
Found the culprit: the generated `jitinterface.h` we use as the wrapper declares this method as `int (__stdcall * appendClassName)(void * thisHandle, CorInfoException** ppException, wchar_t** ppBuf, int* pnBufLen, void* cls, bool fNamespace, bool fFullInst, bool fAssembly);`. There's a `bool`/`BOOL` mismatch.

We need to either fix the generated wrapper code, or our mashalling annotations. Probably former rather than the latter.

Either way, out of scope of this pull request.